src/components/CustomChannelComponent.tsx

```diff
@@ -241,13 +241,13 @@ export function CustomChannelComponent(props: CustomChannelComponentProps) {
     setShowModal(false);
   }
 
-  useScrollOnStreaming({
-    isLastBotMessage,
-    lastMessageRef,
-    // the reply panel component height is about 50px * 3(max replies) = 150px
-    bottomBuffer:
-      dynamicReplyOptions.length > 0 || isStaticReplyVisible ? 150 : 0,
-  });
+  // useScrollOnStreaming({
+  //   isLastBotMessage,
+  //   lastMessageRef,
+  //   // the reply panel component height is about 50px * 3(max replies) = 150px
+  //   bottomBuffer:
+  //     dynamicReplyOptions.length > 0 || isStaticReplyVisible ? 150 : 0,
+  // });
```

Comment:
This hook ensures that the scroll moves to the bottom when messages are streamed. Would it be okay to remove this? I'm also curious if changing the bottomBuffer to `0` might resolve the issue instead

Reply:
@bang9 
Oh, initially it behaved the same even when the bottomBuffer was set to 0, and it also scrolled correctly to the last message when the hook was commented out.

Upon reviewing the hook, I found that it works correctly when the block option of the `scrollIntoView` method is set to 'start' instead of 'end'.

It seems that the settings like 150 and the `scrollUtil()` method at the bottom were used to match the 'start' option.

I am attaching the Before and After videos.

There was an issue where the parent screen slightly scrolled when the first message was received, but now only the chat window inside the iframe scrolls, and it correctly follows when additional messages are received.

Before: https://drive.google.com/file/d/16vOe1LbAB63fxCTydKXNa4BDeCJhh_CV/view?usp=drive_link
After: https://drive.google.com/file/d/1bOTQRi66iHBWZ8A46VuSlqmguYRLzzJQ/view?usp=drive_link

Thanks for your detailed review, Airen:D